composer.json

```diff
@@ -24,6 +24,7 @@
         "league/fractal": "^0.16.0"
     },
     "require-dev": {
+        "php": "^7.0",
```

Comment:
Only the unit tests include 7.1 code, however, it should probably be downgraded to 7.0 to keep the tests and code the same requirements.